pydicom/dataelem.py

```diff
@@ -150,20 +156,22 @@ class DataElement:
     showVR = True
     is_raw = False
 
-    def __init__(self,
-                 tag,
-                 VR,
-                 value,
-                 file_value_tell=None,
-                 is_undefined_length=False,
-                 already_converted=False):
+    def __init__(
+        self,
+        tag : Union[int, str, Tuple[int]],
+        VR: str,
+        value: Any,
+        file_value_tell: Optional[int] = None,
```

Comment:
`file_value_tell` doesn't appear to get used for anything at all. Deprecate and remove?

Reply:
It's used to set `self.file_tell` at line 219, but I guess you mean no calls specify it; I've searched and that appears to be the case.  Yeah, it doesn't seem to have any purpose and very unlikely to be used in external code, so I'm okay with deprecation.

Also I can't find the `ReplaceDataElementValue()` method (or underscore version of the name) referred to in the doc string.  Looks like `file_value_tell` is no longer needed.

Note it is used in `test_equality_class_members` in test_dataelem.py, but is irrelevant because is not used in `__eq__`.

Reply:
Agreed. Seems not to be used since the repo creation.

Reply:
After a bit more work I've realised its the byte offset to the start of the encoded value and gets set when converting from `RawDataElement`. Probably a good idea to keep it.